AdminCore.MailClients.Tests/SMTP/SmtpMailSenderTest.cs

```diff
@@ -0,0 +1,86 @@
+using System.Collections.Generic;
+using AdminCore.DTOs.MailMessage;
+using AdminCore.MailClients.Interfaces;
+using AdminCore.MailClients.SMTP;
+using AdminCore.MailClients.SMTP.Interfaces;
+using AutoFixture;
+using AutoFixture.AutoNSubstitute;
+using NSubstitute;
+using Xunit;
+
+namespace AdminCore.MailClients.Tests.SMTP
+{
+    public class SmtpMailSenderTest
+    {
+        private readonly IFixture _fixture = new Fixture().Customize(new AutoNSubstituteCustomization());
+
+        [Fact]
+        public void SendMessages_CallsSendCorrectNoOfTimesGivenListOfMessages()
```

Comment:
Most of the current unit tests use the naming convention [UnitOfWork_StateUnderTest_ExpectedBehavior].
https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html
e.g
```
Public void Sum_NegativeNumberAs1stParam_ExceptionThrown()
```

Maybe change the names of these to match the same convention as the others?

Reply:
Correction: Most of the service tests use this convention.